.storybook/theme/utils/global-theme-controller.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {Fragment, useEffect} from 'react';
+import {Theme, useTheme} from '../../../src';
+
+export interface GlobalThemeControllerProps {
+    children?: React.ReactNode;
+}
+
+export const GlobalThemeController = ({children}: GlobalThemeControllerProps) => {
+    const theme = useTheme();
+    const [prevTheme, setPrevTheme] = React.useState(theme);
+
+    const updateBodyClassName = (newTheme: Theme) => {
+        const bodyEl = document.body;
+
+        if (!bodyEl.classList.contains('yc-root')) {
+            bodyEl.classList.add('yc-root');
+        }
+
+        bodyEl.classList.toggle('yc-root_theme_light', newTheme === Theme.Light);
+        bodyEl.classList.toggle('yc-root_theme_dark', newTheme === Theme.Dark);
+    };
+
+    useEffect(() => {
+        if (theme !== prevTheme) {
+            updateBodyClassName(theme);
+        }
+
+        return () => {
+            if (theme !== prevTheme) {
+                setPrevTheme(theme);
+            }
+        };
+    }, [theme, prevTheme]);
+
+    useEffect(() => {
```

Comment:
is it possible to join this two `useEffect`?

Reply:
I think no, we can't. As `g-root` class not set initially, the second `useEffect` sets it for first time. The first `useEffect` calls used only when theme changes.
But one can set `prevTheme` to, e.g. `null`, but it looks more unobvious for me